### ISSUE TITLE
Remove thumbnail image when "Delete local avatar" chosen

### ISSUE THREAD

**Issue by kblizeck** [10up/simple-local-avatars#55]
**Is your enhancement related to a problem? Please describe.**
Once an avatar is loaded, if I wish to remove the avatar, I don't receive any notification that removing it (clicking "Delete local avatar") worked. It does remove it, but the thumbnail doesn't disappear on click, and the Update Profile button remains disabled.

**Describe the solution you'd like**
Remove thumbnail of previous Avatar once "Delete local avatar" has been clicked.

**Designs**

**Describe alternatives you've considered**

**Additional context**
https://share.getcloudapp.com/Blu5BJ50

**Comment by jeffpaul** [comment]
@kblizeck I just tested myself with only Simple Local Avatars installed and it works as you described in the desired solution.  Can you share the Site Health Info from your site?  It seems like maybe there's a conflicting plugin or something else that may be causing this issue for you.

**Comment by kblizeck** [comment]
Hi @jeffpaul - here's my Site Health Info

```
### wp-core ###

version: 5.4.2
site_language: en_US
user_language: en_US
timezone: +00:00
permalink: /%year%/%monthnum%/%day%/%postname%/
https_status: false
user_registration: false
default_comment_status: closed
multisite: true
user_count: 2
site_count: 1
network_count: 1
dotorg_communication: true

### wp-active-theme ###

name: Client
version: 0.1.0
author: HoG
author_website: https://houseofgiants.com
parent_theme: none
theme_features: automatic-feed-links, title-tag, post-thumbnails, html5, editor-color-palette, menus
theme_path: /Users/username/Local Sites/client/app/public/wp-content/themes/client

### wp-themes-inactive (1) ###

Twenty Twenty: version: 1.4, author: the WordPress team

### wp-plugins-active (1) ###

Simple Local Avatars: version: 2.1.1, author: Jake Goldman, 10up

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
gd_version: bundled (2.1.0 compatible)
ghostscript_version: not available

### wp-server ###

server_architecture: Darwin 19.5.0 x86_64
httpd_software: nginx/1.16.0
php_version: 7.3.5 64bit
php_sapi: fpm-fcgi
max_input_variables: 4000
time_limit: 1200
memory_limit: 256M
max_input_time: 600
upload_max_size: 300M
php_post_max_size: 1000M
curl_version: 7.54.0 LibreSSL/2.6.5
suhosin: false
imagick_availability: false

### wp-database ###

extension: mysqli
server_version: 8.0.16
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /Users/username/Local Sites/client/app/public/wp-content
WP_PLUGIN_DIR: /Users/username/Local Sites/client/app/public/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable
```

**Comment by jeffpaul** [comment]
@kblizeck I just ran another local test, relatively close to your setup (see my Site Health Info below) and things worked as expected.  Can you check and see if there are any console errors that display on that page?

```
### wp-core ###

version: 5.4.2
site_language: en_US
user_language: en_US
timezone: +00:00
permalink: /%postname%/
https_status: false
user_registration: 0
default_comment_status: open
multisite: false
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /Users/jeffpaul/Local Sites/sla/app/public
wordpress_size: 39.89 MB (41829423 bytes)
uploads_path: /Users/jeffpaul/Local Sites/sla/app/public/wp-content/uploads
uploads_size: 16.12 KB (16509 bytes)
themes_path: /Users/jeffpaul/Local Sites/sla/app/public/wp-content/themes
themes_size: 1.58 MB (1656307 bytes)
plugins_path: /Users/jeffpaul/Local Sites/sla/app/public/wp-content/plugins
plugins_size: 226.20 KB (231626 bytes)
database_size: 2.11 MB (2211840 bytes)
total_size: 43.82 MB (45945705 bytes)

### wp-active-theme ###

name: Twenty Twenty (twentytwenty)
version: 1.5
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: automatic-feed-links, custom-background, post-thumbnails, custom-logo, title-tag, html5, align-wide, responsive-embeds, customize-selective-refresh-widgets, editor-color-palette, editor-font-sizes, editor-styles, widgets, menus, editor-style
theme_path: /Users/jeffpaul/Local Sites/sla/app/public/wp-content/themes/twentytwenty

### wp-plugins-active (2) ###

Simple Local Avatars: version: 2.1.1, author: Jake Goldman, 10up
WP Downgrade | Specific Core Version: version: 1.2.1, author: Reisetiger

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
gd_version: bundled (2.1.0 compatible)
ghostscript_version: not available

### wp-server ###

server_architecture: Darwin 19.5.0 x86_64
httpd_software: nginx/1.16.0
php_version: 7.3.5 64bit
php_sapi: fpm-fcgi
max_input_variables: 4000
time_limit: 1200
memory_limit: 256M
max_input_time: 600
upload_max_size: 300M
php_post_max_size: 1000M
curl_version: 7.54.0 LibreSSL/2.6.5
suhosin: false
imagick_availability: false
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 8.0.16
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /Users/jeffpaul/Local Sites/sla/app/public/wp-content
WP_PLUGIN_DIR: /Users/jeffpaul/Local Sites/sla/app/public/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable
```

**Comment by kblizeck** [comment]
@jeffpaul There are no console.log errors. Are you running a multisite setup?

**Comment by jeffpaul** [comment]
@kblizeck no, I wasn't, I missed that in your info.  Are you using sub-directories or folders?  Is that user in the initially created site or the second one created?

**Comment by kblizeck** [comment]
@jeffpaul I'm sorry - I didn't include that in the original issue!

- Multisite install - subdomain setup
- The same issue occurs when editing the Avatar either through the Network Admin - Users, or through the initially created site Dashboard Users. 
- I only have one site installed currently, as we haven't started development on the second.
This

**Comment by jeffpaul** [comment]
@kblizeck I just tested this in a multisite subdomain setup and have not been able to replicate your issue.  Mind sharing the image you're testing with in case there's something about it?  I also tried to replicate with a user with `@` and `.` in the username to even more closely match your screencast, but WP limits usernames to only lowercase letters and numbers so assuming you're creating that user via an alternate method you might try with a user without those `@` and `.` characters and see if that fixes things (in case there's something going related there.
